creature.py

```diff
@@ -11,44 +27,58 @@ def __init__(self, x, y, color, energy):
         self.x = x
         self.y = y
         self.color = color
+        self.actioncolor = BLACK
         self.energy = energy
         self.id = Creature.id_count
         Creature.id_count += 1
 
-    def step(self, world, nearby):
+
+    def step(self, nearby):
         if self.energy > 500:
+            self.actioncolor = YELLOW
             return None
-        dx = 0
-        dy = 0
-
-        dist = self.distance(0, 0)
-        if dist > world.size - self.radius():
-            dx -= 0.05 * self.x / world.size
-            dy -= 0.05 * self.y / world.size
-
-        for other in nearby:
-            dist = self.distance(other.x, other.y)
-            if dist == 0:
-                print(dist)
-            if isinstance(other, Creature) and other.energy > self.energy:
-                weight = -2
+
+        flee = eat = 0
+        slice_preference = np.zeros(SLICES)
+        for other, slice, distance in nearby:
```

Comment:
maybe pick a different variable name for slice seeing that it is a built-in method?

Reply:
Good point. Any better name? pie?